projects/packages/forms/src/dashboard/inbox/stage/index.js

```diff
@@ -227,11 +226,7 @@ export default function InboxView() {
 		}
 
 		// Update sidebar if item changed or needs refresh
-		if (
-			! sidePanelItem ||
-			getItemId( sidePanelItem ) !== getItemId( recordToShow ) ||
-			sidePanelItem !== recordToShow
```

Comment:
I was trying to understand what this line does, but I couldn't see it affect anything. 

Was it there to hide the sidebar when you move a response visible in the sidebar to spam/trash?